data/brands/shop/convenience.json

```diff
@@ -1270,6 +1270,18 @@
         "shop": "convenience"
       }
     },
+    {
+      "displayName": "Dali",
+      "locationSet": {"include": ["ph"]},
+      "tags": {
+        "brand": "Dali",
+        "brand:website": "https://dali.ph",
+        "brand:wikidata": "Q119854219",
+        "name": "Dali",
+        "shop": "convenience",
+        "strapline":"Everyday grocery"
```

Comment:
```suggestion
```
NSI does not document any content used for commercial promotion

Reply:
Noted. Thanks.